### PR TITLE
1.18-exp6 stuff

### PR DESCRIPTION
**Pull request by apple502j**
Some shared constants and biome stuff.

### CODE REVIEW

File: mappings/net/minecraft/world/biome/source/util/VanillaBiomeParameters.mapping [hunk 1]
```diff
@@ -77,6 +77,11 @@ CLASS net/minecraft/class_6461 net/minecraft/world/biome/source/util/VanillaBiom
 		ARG 1 parameters
 	METHOD method_37718 writeRiverBiomes (Lcom/google/common/collect/ImmutableList$Builder;Lnet/minecraft/class_6452$class_6454;)V
 		ARG 1 parameters
+	METHOD method_37846 getShoreOrNearInlandBiome (IILnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_5321;)Lnet/minecraft/class_5321;
```
Comment (liach) [inline mappings/net/minecraft/world/biome/source/util/VanillaBiomeParameters.mapping:80]: This one really checks special cases and uses shattered savanna than the fallback in those cases.

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/world/biome/source/util/VanillaBiomeParameters.mapping (hunk 1 above)
Comment (mschae23) [reply mappings/net/minecraft/world/biome/source/util/VanillaBiomeParameters.mapping:80]: I think I always named it something like `getBiomeOrShatteredSavanna` in my PRs.

Comment (liach) [reply mappings/net/minecraft/world/biome/source/util/VanillaBiomeParameters.mapping:80]: Yeah, out of all its usages, one of it is for picking a biome for shore and near-inland; a few others are near-inland, and one usage is like make a call and never made use of the result biome

**Comment by apple502j** [comment]
I'll probably make a new PR with just some SharedConstants edit. Biome stuff is full of weird stuff and weirdness stuff...

**Pull request closed without merging**